packages/web/src/components/nav-banner/NavBanner.tsx

```diff
@@ -26,15 +29,10 @@ type NavBannerProps = {
 }
 
 const NavBanner = (props: NavBannerProps) => {
-  const {
-    tabs,
-    onSortByRecent,
-    onSortByPopular,
-    shouldMaskContent,
-    dropdownDisabled,
-    isArtist,
-    empty
-  } = props
+  const { tabs, onSortByRecent, onSortByPopular, dropdownDisabled, isArtist } =
+    props
+
+  const hideSortButton = useMediaQuery('(max-width: 1200px)')
```

Comment:
do we want to document these values? maybe in layout.ts?

Reply:
I can add it. This was kind of a trial and error to get it to disappear at a reasonable point. I think this needs some more work later on because the experience already breaks down at this width and hiding it only give you an extra 24px or so before it starts crunching up again 🤷